src/.env

```
@@ -1 +1 @@
-VITE_API_URL='https://api.amrti.com/'
\ No newline at end of file
+VITE_API_URL='http://api.amrti.com'
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)